### ISSUE TITLE
Windows HCM: Switch

### ISSUE THREAD

**Issue by gfellerph** [swisspost/common-web-frontend#82]
![image](https://user-images.githubusercontent.com/1659006/165494485-c9f23648-8a04-4ebb-ad39-89018f0b57d7.png)

<p>Switch should have a border in none checked Version to indicate that's a switch.</p>

<p>Wrong color is used in HCM. Correct color would be highlight:</p>

<p><a href="https://www.w3.org/TR/css-color-4/#css-system-colors">CSS Color Module Level 4</a>&nbsp;</p>
